step-function/get-files/test_get_files.py

```diff
@@ -0,0 +1,113 @@
+from os import environ
+
+import pytest
+from botocore.stub import Stubber
+from src.main import S3_CLIENT, get_expiration_time, get_object_file_type, lambda_handler
+
+
+@pytest.fixture(autouse=True)
+def setup_env():
+    environ['BUCKET'] = 'mybucket'
+    environ['AWS_REGION'] = 'region'
```

Comment:
use `monkeypatch` here?  https://docs.pytest.org/en/stable/monkeypatch.html
```
def setup_env(monkeypatch):
    monkeypatch.setenv('BUCKET', 'mybucket')
    monkeypatch.setenv('AWS_REGION', 'region')
```

Reply:
and does setting `AWS_REGION` that way actually work?  I wouldn't expect `region` to be a valid value.  I worry your environment might be pulling the value from your `~/.aws/config` file, which had happened to me before.